recommendations/tech_learning.md

```diff
@@ -46,65 +58,95 @@ This is a living document; if you find something out of date or have an addition
-| [TypeScript Deep Dive](https://basarat.gitbooks.io/typescript/) | Get to the part where TypeScript feels good quicker |
-| []() |  |
+| Name                                              | Description                                                                                                              |
+| ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| [[A.] Javascriptures][javascriptures]             | A series of Artsy workshops covering our 2018 frontend stack: Js, TypeScript, React, Relay and more                      |
+| [[A.] Is GraphQL the future?][is_graphql]         | Thoughts from Artsy Alum Alan Johnson                                                                                    |
+| [Getting Started- React][getting_started_react]   | From the official react docs                                                                                             |
+| [React Patterns][react_patterns]                  | A rundown on clean & effective react syntax                                                                              |
+| [How to pass props to components in React][props] | Key to understanding React as a whole                                                                                    |
+| [Learn GraphQL][learn_gql]                        | "GraphQL is a query language for your API" - specify only the data you need in the shape that works for your application |
+| [Relay][relay]                                    | Define your components' api dependencies in the same file and let relay request them automatically via GraphQL           |
+| [Redux - Egghead.io][redux_egghead]               | [subscription required] An introduction to Redux with creator Dan Abramov                                                |
+| [A Gist about Jest and Enzyme][jest_enz]          | Difference between `shallow`, `mount` and `render` of Enzyme                                                             |
+| [TypeScript Deep Dive][ts_dive]                   | Get to the part where TypeScript feels good quicker                                                                      |
+| []()                                              |                                                                                                                          |
 
 ### Frontend iOS Practice
-*See also [Javascript](#javascript-+-nodejs) & [Swift + Objective C](#swift-+-objective-c)*
 
-| Name | Description |
-| -- | -- |
-| [React Native](https://facebook.github.io/react-native/) | The official docs |
-| [Expo](https://expo.io/) | 'A free and open source toolchain built around React Native to help you build native iOS and Android projects using JavaScript and React' |
```

Comment:
I did remove expo though, because we don't use it anywhere and probably won't ever need it.